docs/docs/tracker-stores.mdx

```diff
@@ -394,3 +394,20 @@ extending the Rasa image to include the module, or mounting the module as volume
    a_parameter: a value
    another_parameter: another value
  ```
+
+## Fallback Tracker Store
+
+In case the primary tracker store configured in `endpoints.yml` becomes unavailable, the rasa agent will issue an
+error warning and fall back on the `InMemoryTrackerStore` implementation. A new dialogue session will be started for
```

Comment:
Should this be "error message" instead?

Reply:
It's a log error message indeed , I will amend 👍🏻